.gitignore

```diff
@@ -1,4 +1,3 @@
-.idea/
-
 node_modules/
+
```

Comment:
Consider adding .idea/workspace.xml file to gitignore, it contain user-specific settings only.